doc/sphinx/source/recipes/recipe_gier20bg.rst

```diff
@@ -193,6 +193,7 @@ Example plots
    Barplot of the growth rate, averaged over all years, with standard deviation
    of interannual variability.
 
+.. gallery=True
```

Comment:
If we want this, it's probably better to use the following syntax:

```rst
..
  gallery=True
```

See https://stackoverflow.com/a/57267316

Reply:
done!